Mathlib/MeasureTheory/MeasurableSpace/Basic.lean

```diff
@@ -1630,12 +1641,18 @@ def piMeasurableEquivTProd [DecidableEq δ'] {l : List δ'} (hnd : l.Nodup) (h :
   measurable_invFun := measurable_tProd_elim' h
 #align measurable_equiv.pi_measurable_equiv_tprod MeasurableEquiv.piMeasurableEquivTProd
 
+variable (π) in
+/-- The measurable equivalence `(∀ i, π i) ≃ᵐ π ⋆` when the domain of `π` only contains `⋆` -/
+@[simps! (config := .asFn)]
```

Comment:
I've never seen this `.asFn` before; should we be using it more often?

Reply:
Sure, we can use it by default: #8296